src/libCps/CpsRunEvm.cpp

```diff
@@ -616,7 +697,15 @@ void CpsRunEvm::ProvideFeedback(const CpsRun& previousRun,
       }
     }
   } else {
-    // TODO: allow scilla runner to provide feedback too
+    const auto& scillaResult = std::get<ScillaResult>(results.result);
```

Comment:
Can't tell from reading the code, but does this not zap the logs from the scilla run, so that scilla events/messages don't appear in the transaction receipt?